digitalearthau/coherence.py

```diff
@@ -54,7 +55,7 @@ def main(expressions, check_locationless, archive_locationless, check_ancestors,
     """
     global _siblings_count
     uiutil.init_logging()
-    config_file = test_dc_config if test_dc_config else ''
+    config_file = test_dc_config if test_dc_config else os.environ.get('DATACUBE_CONFIG_PATH')
```

Comment:
This is still potentially fragile. It would be better to set `config_file` to `None` and allow the default configuration discovery process to run.

(or just delete this line and make sure that `test_dc_config` defaults to `None` and use it directly)

Reply:
Updated as per review comments by removing `config_file` variable assignment line and defaulting `test_dc_config` to `None`.